README.md

```diff
@@ -4,6 +4,8 @@ Instructions on how to instantiate a multi-node, IPv6-only Kubernetes cluster us
 # Overview
 So you'd like to take Kubernetes IPv6 for a test drive, or perhaps do some Kubernetes IPv6 development? The instructions below describe how to bring up a multi-node, IPv6-only Kubernetes cluster that uses the CNI bridge and host-local IPAM plugins, using kubeadm to stand up the cluster.
 
+If you are too lazy, you may want to have a look at the automated [vagrant](vagrant) environment.
```

Comment:
@Oryon (Pierre), I just added a "Quick Start Options" section, so that I could also include the Kube-in-the-Box (DinDinD) option. I left a "TBD" bullet item for your Vagrant setup. Would you mind rebasing your Vagrant PR to this new section? Thanks!!!

Reply:
Done. :-)